homepage/models.py

```diff
@@ -322,6 +322,20 @@ def get_courses_followed_by_app_user(app_user):
         # get only 'course' elements from user_course_pair elements
         return [user_course_pair.course for user_course_pair in pairs]
 
+    @classmethod
+    def is_following_course(cls, user, course):
+        return True if course in cls.get_courses_followed_by_app_user(user.appuser) else False

```

Comment:
```suggestion
        return course in cls.get_courses_followed_by_app_user(user.appuser)

```

Reply:
@kobihk 
My bad, missed it in the previous comment.
Fixed